apis/s3/v1alpha2/bucketpolicy_types.go

```diff
@@ -28,16 +28,15 @@ type BucketPolicyParameters struct {
 	// +immutable
 	Region string `json:"region"`
 
-	// This is the current IAM policy version
-	Version string `json:"version"`
-
-	// This is the policy's optional identifier
-	// +immutable
+	// JSONStatement is a stringified version of the S3 Bucket Policy.
+	// either jsonBody or policyBody must be specified in the policy
 	// +optional
-	ID string `json:"id,omitempty"`
+	JSONBody *string `json:"jsonBody,omitempty"`
```

Comment:
What do you think about `rawPolicy` and `policy` field names? I feel like `Body` could be interpreted as only the `Statements` object in the JSON that doesn't include `Version`, but we need `Version` in this string.

Reply:
Should be done now! :)